tests/integration_tests/validation/conformance_suite_configurations/xbrl_calculations_1_1.py

```diff
@@ -0,0 +1,40 @@
+from pathlib import PurePath
+from tests.integration_tests.validation.conformance_suite_config import ConformanceSuiteConfig
+
+config = ConformanceSuiteConfig(
+    args=[
+        '--plugin', 'loadFromOIM',
+        '--plugin', '../../arelle/examples/plugin/testcaseCalc11ValidateSetup.py',
+    ],
+    expected_failure_ids=frozenset([
+        # The loadFromOIM plugin is required to load the conformance suite json
+        # files. However, OIM validation raises xbrlxe:unsupportedTuple for
+        # tuples which then raises calc11e:tuplesInReportWarning in calc 1.1.
+        # This isn't modeled in the conformance suite, but expected if both
+        # calc 1.1 and OIM validation are performed together.
+        # https://www.xbrl.org/Specification/xbrl-xml/REC-2021-10-13/xbrl-xml-REC-2021-10-13.html#unsupportedTuple
+        # https://www.xbrl.org/Specification/calculation-1.1/REC-2023-02-22/calculation-1.1-REC-2023-02-22.html#error-calc11e-tuplesinreportwarning
```

Comment:
As a followup, is it reasonable to differentiate warnings from errors in `expected_failure_ids`, or to allow enumerating the unexpected warnings instead?

Reply:
Yes. I'll write up an issue